spec.html

```diff
@@ -11892,7 +11892,7 @@
           1. Let _tailCall_ be IsInTailPosition(_thisCall_).
           1. Return ? EvaluateDirectCall(_func_, _thisValue_, |Arguments|, _tailCall_).
         </emu-alg>
-        <p>A |CallExpression| evaluation that executes step 3.a.vi is a <dfn>direct eval</dfn>.</p>
+        <p>A |CallExpression| evaluation that executes step 3.a.vii is a <dfn>direct eval</dfn>.</p>
```

Comment:
> Or, consider external specs linking to steps. In order to do so they would have to first create a PR adding marker, but I'd probably reject it because I'd have no way of knowing whether it was still needed or not.

You could require that such a PR include a mechanism (e.g., a URL) by which to determine whether the marker is still needed.